.github/CODEOWNERS

```diff
@@ -100,12 +100,12 @@
 /lib/internal/bootstrap/loaders.js @nodejs/modules
 /src/module_wrap* @nodejs/modules @nodejs/vm
 
-# N-API
+# Node.js API 
```

Comment:
"Node.js API" seems like it will inevitably cause confusion. Is this the name that reached consensus in discussion? (I can't find the issue in the tracker. Maybe I'm looking in the wrong repository?)

Reply:
@Trott I believe the name is still open for discussion at https://github.com/nodejs/abi-stable-node/issues/420.

Reply:
node-api was what was being discussion to replace n-api and napi.